apps/prairielearn/src/sprocs/assessment_instances_regrade.sql

```diff
@@ -31,8 +30,7 @@ BEGIN
     -- first update Homeworks
     updated := FALSE;
     IF assessment_type = 'Homework' THEN
-        SELECT *
-        INTO updated, new_instance_question_ids
+        SELECT updated INTO updated
```

Comment:
This is not directly related, but I detected this as unused while removing the other parameter with the same name, so I removed it as well.